libbeat/cmd/instance/metrics.go

```diff
@@ -58,6 +59,9 @@ func setupMetrics(name string) error {
 	monitoring.NewFunc(beatMetrics, "memstats", reportMemStats, monitoring.Report)
 	monitoring.NewFunc(beatMetrics, "cpu", reportBeatCPU, monitoring.Report)
 	monitoring.NewFunc(beatMetrics, "runtime", reportRuntime, monitoring.Report)
+	if runtime.GOOS == "linux" {
+		monitoring.NewFunc(beatMetrics, "cgroup", reportBeatCgroups, monitoring.Report)
+	}
```

Comment:
How about moving this block into the `setupPlatformSpecificMetrics` function?

Reply:
Done